gap/FunctorCategories.gi

```diff
@@ -1231,96 +1231,28 @@ InstallMethodWithCache( Hom,
       SetIsAbelianCategoryWithEnoughInjectives( Hom, true );
       
       AddIsProjective( Hom,
-        function ( F )
-          local iso;
-          
-          iso := IsomorphismOntoCategoryOfQuiverRepresentations( Hom );
-          
-          return IsProjectiveRepresentation( ApplyFunctor( iso, F ) );
-          
-      end );
+        F -> IsIsomorphism( ProjectiveCover( F ) )
```

Comment:
To comply with the specification and not with the implementation I would replace this line by

```
  F -> IsSplitEpimorphism( ProjectiveCover( F ) )
```